docker-compose.debug.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+services:
+  app:
+    build: .
+    env_file: env_file
+    volumes:
+      - ckan:/usr/lib/ckan
+      - ./config:/etc/ckan
+      - .:/opt/inventory-app
+      - ../ckanext-s3filestore/:/usr/lib/ckan/src/ckanext-s3filestore/
```

Comment:
I don't think we want to commit this... maybe as a comment?

Reply:
ckanext-s3filestore isn't a good example since you need to have the keys to make that work. Agree this would be better as a comment for someone to uncomment.

Reply:
Moved to comment and generalized.